repo2docker/app.py

```diff
@@ -264,6 +264,20 @@ def get_argparser(self):
             help='Do not run container after it has been built'
         )
 
+        argparser.add_argument(
+            '--publish',
```

Comment:
Does this automatically make -p work or does that need to be explicitly added?

Reply:
-p must be explicitly added and will add it now and also rebase the code.